ubtue_Collectanea Christiana Orientalia.js

```diff
@@ -0,0 +1,177 @@
+{
+	"translatorID": "f24cd419-5892-450a-b39e-be2ff3191757",
+	"label": "Collectanea Christiana Orientalia",
+	"creator": "Timotheus Kim",
+	"target": "/issue/view/",
```

Comment:
target seems to be too generic for a translator which is meant to be used only for 1 specific journal